tests/installation/registration/register_via_rmt.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the system via local RMT in the installer with server url
+# and enabling update repositories.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    $testapi::distri->get_registration()->register_via_rmt({
+            server => get_var('RMT_SERVER')});
```

Comment:
we can harcode it, it is exposed everywhere, progress ticket, bugs, etc, beside that we avoid to use settings as much as possible, otherwise the setting needs to be documented in `variables.md`, but in this case doesn't worth it.

Reply:
I can do that, but since this will be it's own testsuite, maybe it makes more sense to put it in a dedicated YAML file for that testcase instead of hardcoding it in the code? Mixing data and logic that much just doesn't sit very well with me. 
  
like this one for example https://github.com/BillAnastasiadis/os-autoinst-distri-opensuse/blob/cb0480bb229e030731d52fd8460cf2d121249477/schedule/yast/select_modules_and_patterns/select_modules_and_patterns.yaml# (but for the new testsuite, `register_with_rmt` or whatever it will be named)

Just wondering. If you don't like the idea, I'll just hardcode it and be done with it :p

Reply:
the problem is that there is a lack of control on those variables and it is better to not contribute to that mess. In order to not hardcode it the best solution is using testdata, you have seen probably in many test that approach, but in the past we noticed that it was unnecessary for this kind of things that are sort of constants. We also agreed in the past that on the layer Test doesn't hurt, at the level of Page or Controller would be a problem and not the best code style. If you please could hardcode it for now, perhaps all these ideas would make more sense to you in a while.

Reply:
Maybe the testdata approach makes sense, containing the server and all the repos to be validated (as there is already a `validate_repos` function that uses testdata). I'm going for that and we can evaluate it. Thanks for the suggestions!

Reply:
ok, let's keep it then, thanks for your input @JRivrain, @BillAnastasiadis could you please document the variable in `variables.md`.

Reply:
Ok, will do

Reply:
There exist an setting already used in current migration test cases, SMT_URL, this setting used to flag SMT or RMT url. 
But this setting seems not mentioned in the [variables.md]

I list one smt test case and one rmt test case for example:
https://openqa.suse.de/tests/8777824#step/yast2_migration/1
https://openqa.suse.de/tests/8776608#settings